spec/unit/readerview_spec.lua

```diff
@@ -47,8 +47,9 @@ describe("Readerview module", function()
         end
     end)
 
-    -- TODO FIX THESE TESTS THEY'RE BROKEN!!!
```

Comment:
To be clear, the test wasn't broken per se, but it was secretly depending on some other test turning off the footer.